frontend/src/__tests__/cypress/cypress/pages/resources.ts

```diff
@@ -0,0 +1,178 @@
+import { Contextual } from './components/Contextual';
+
+class Resources {
+  visit() {
+    cy.visit('/resources');
+    this.wait();
+  }
+
+  private wait() {
+    cy.findByTestId('app-page-title');
+    cy.testA11y();
+  }
+
+  findEmptyState() {
+    return cy.findByTestId('empty-state-title');
+  }
+
+  findClearFilterButton() {
+    return cy.findByTestId('clear-all-filters');
+  }
+
+  getCardView() {
+    return new CardView(() => cy.findByTestId('learning-center-card-view'));
+  }
+
+  getListView() {
+    return new ListView(() => cy.findByTestId('learning-center-list-view'));
+  }
+
+  getLearningCenterFilters() {
+    return new LearningCenterFilters(() => cy.findByTestId('learning-center-filters'));
+  }
+
+  getLearningCenterToolbar() {
+    return new LearningCenterToolbar(() => cy.findByTestId('learning-center-toolbar'));
+  }
+
+  getQuickStartDrawer() {
+    return new QuickStartDrawer(() => cy.findByTestId('qs-drawer-creatingAJupyterNotebook'));
+  }
+
+  findLeaveQuickStartDialogue() {
+    return cy.findByRole('dialog');
+  }
+
+  findLeaveButton() {
+    return cy.findByRole('button', { name: 'Leave' });
+  }
+}
+
+class CardView extends Contextual<HTMLElement> {
+  findCardItems() {
+    return this.find().findAllByTestId(['card']);
+  }
+
+  findCard(id: string) {
```

Comment:
This returns `return this.find().findByTestId(['card', id]);` which is a selector. 

Using `find` here is correct. Unless the function returns a `new PageObject`  in which case `get` should be used.

Reply:
That being said I think I know where @pnaik1 is going with this.

I would expect `getCard(id: string)` to return a new page object which has all the selectors scoped to that code instead of having the `CardView` parameterize each method to target a specific card.

eg.
```
getCard(id: string) {
  return new Card(() => this.find().findByTestId(['card', id]));
}

class Card extends Contextual {
...
findSomethingOnACard() { ... }
}
```

Reply:
That is precisely what I was indicating. 
We do indeed have a generic Card component in place, Can we make use of that if possible?